cljfmt/test/cljfmt/test_util/cljs.cljc

```diff
@@ -1,5 +1,5 @@
 (ns cljfmt.test-util.cljs
-  (:require [clojure.test :as test]
+  (:require [cljs.test :as test]
```

Comment:
It turned out that `cljs.test` is loaded here during macro-expansion, so it really needed to be `cljs.test` after all.